chart/templates/deployment.yaml

```diff
@@ -44,13 +44,13 @@ spec:
           livenessProbe:
             httpGet:
               path: /health
-              port: 10999
+              port: 8080
             initialDelaySeconds: 0
             periodSeconds: 5
           readinessProbe:
             httpGet:
-              path: /health
-              port: 10999
+              path: /ready
+              port: 8080
```

Comment:
Please leave those changes as they are (for now :) )

Add the respective probes to the argo-service container.